src/Microsoft.Sbom.Targets/SbomInputValidator.cs

```diff
@@ -85,16 +85,22 @@ public EventLevel ValidateAndAssignVerbosity()
         switch (this.Verbosity.ToLower().Trim())
         {
             case "verbose":
+                this.Verbosity = "Verbose";
```

Comment:
Adding these meant that the `Verbosity` member on the `GenerateSbom` Task could be checked in tests to verify postconditions more easily than scraping logs.